app/cdap/components/GridTable/types.ts

```diff
@@ -48,3 +48,36 @@ export interface IHeaderNamesList {
 export interface IType {
   [key: string]: string | number;
 }
+
+export interface IObject {
+  [key: string]: string | number | IRecords | boolean;
```

Comment:
This type doesn't provide any value

Reply:
We require this interface for the API data which contain key value pairs where key will always be a string but values can differ.